### ISSUE TITLE
Security: Use of strncopy() function in "Get Started" project

### ISSUE THREAD

**Issue by prafullkotecha** [Microsoft/azure-iot-developer-kit#322]
Here's the message from devskim plugin:
```
[Devskim: Finding DS111237]
Banned C function detected (strncpy)
Severity: [Important]

strncpy is dangerous, as if the source contains 'n' or more characters, it will not null terminate the destination.

Fix Guidance: Use strcpy_s or strlcpy if possible. If no safe function is viable, strcpy/strncpy should be proceeded by conditional checks to verify tha that the source string will fit in the destination with a null termnator.

More Info:
https://github.com/Microsoft/DevSkim/blob/master/guidance/DS111237.md
#define strncpy(dest, src, len) __builtin___strncpy_chk(dest, src, len, __builtin_object_size(dest, 1))
```

**Comment by ArthurMa1978** [comment]
This issue was moved to Microsoft/devkit-sdk#764

**Comment by ArthurMa1978** [comment]
Thank you Prafull, moved this issue to devkit-sdk repo and will be fixed in 1.3.3
